### DISCUSSION TITLE
merge request update (plus pipeline runs) even if not scheduled or if rate limited

### DISCUSSION THREAD

**Discussion by vquie** [renovatebot/renovate#21124] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.14.3

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### If you're self-hosting Renovate, tell us what version of the platform you run.

15.11.0-pre

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I would like to see that renovate respects the schedules and rate limits even when keeping merge requests up to date.

### What you tried so far.

1. `"rebaseWhen": "conflicted"`
2. `"extends": ["schedule:earlyMondays"]`

next thing is disable renovate

### Relevant debug logs

I don't think there are logs needed as it is doing its job even if it (from my perspective) shouldn't.

**Comment by rarkins** [comment, marked as answer]
Configure `"updateNotScheduled": true`

**Reply by vquie** [reply]
It's right there in the docs and there is also a preset. Thanks.
